hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/launcher/ContainerRelaunch.java

```diff
@@ -115,6 +116,14 @@ public Integer call() {
           .setContainerLocalDirs(containerLocalDirs)
           .setContainerLogDirs(containerLogDirs)
           .build());
+    } catch (ConfigurationException e) {
+      LOG.error("Failed to relaunch container.", e);
```

Comment:
It will be redundant, since the exception type is usually visible, but I fixed it.